lib/utils/formatOptions.js

```diff
@@ -0,0 +1,19 @@
+// @flow
```

Comment:
`utils` is also starting to get kinda big, so I think a `utils/` folder with each individual function in a file is a little more manageable.